django/contrib/messages/storage/cookie.py

```diff
@@ -163,3 +163,12 @@ def _decode(self, data):
         # with the data.
         self.used = True
         return None
+
+    def __repr__(self):
+        return '<%(cls)s [%(count)d] used=%(used)r, added_new=%(added)r, cookie_name="%(name)s">' % {
```

Comment:
Maybe this should contain something like `[num_messages=%(count)d]` -- otherwise, I don't think it's intuitive what the number in brackets is.